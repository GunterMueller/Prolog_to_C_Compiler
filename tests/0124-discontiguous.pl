:- discontiguous foo/1, bar/0, bar/1.

foo(1).
bar :- display(bar1), nl.

main :- foo(X), write(X), nl, fail.
main :- bar, bar(123).

foo(2).
bar(X) :- display(X), nl.
foo(3).
