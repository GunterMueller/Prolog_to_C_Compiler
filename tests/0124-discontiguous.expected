1
2
3
bar1
123
